common/app/model/content/Atoms.scala

```diff
@@ -53,6 +54,16 @@ object MediaAssetPlatform extends Enum[MediaAssetPlatform] with PlayJsonEnum[Med
   case object Url extends MediaAssetPlatform
 }
 
+sealed trait MediaWrapper extends EnumEntry
+
+object MediaWrapper extends Enum[MediaWrapper] with PlayJsonEnum[MediaWrapper] {
+  val values = findValues
+
+  case object MainMedia extends MediaWrapper
+  case object ImmersiveMainMedia extends MediaWrapper
+  case object EmbedPage extends MediaWrapper
```

Comment:
Unfortunately, as the fronts are burned in on S3 it is not easy to add mandatory fields to models that are used by Facia.